pkgs/development/libraries/libgsignon-glib/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchFromGitLab, pkgconfig, automake, autoconf, libtool
+, glib, vala, gobjectIntrospection, gsignond, gettext, check
+, dbus, python3Packages, glibcLocales, docbook_xml_dtd_43
+, docbook_xml_dtd_45, docbook_xsl, gtk-doc, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "libgsignon-glib";
+  version = "2.4.1";
+
+  name = "${pname}-${version}";
+
+  src = fetchFromGitLab {
+    owner = "accounts-sso";
+    repo = pname;
+    rev = version;
+    sha256 = "1h7s5ibvy0pbk9pm95a6gm293cpcqq65daszi6gl31i64mc758ap";
+  };
+
+  outputs = [ "out" "dev" "devdoc" "py" ];
+
+  nativeBuildInputs = with python3Packages; [
+    autoconf
+    automake
+    check
+    gettext
+    glibcLocales
+    gobjectIntrospection
+    libtool
+    pkgconfig
+    vala
+    python
```

Comment:
You should use either `python3` and `python3.pkgs`, or `python3Packages.python` and `python3Packages`. In the worst case, those two variants can point to a different packages.

Reply:
Also no need to use `with` construct for just a single package.

Reply:
yeah no with.

Reply:
I'm using python3Packages.python and the same for python3Packages.pygobject3 already.